tests/SprykerEcoTest/Client/FactFinderNg/codeception.yml

```diff
@@ -0,0 +1,24 @@
+namespace: SprykerEcoTest\Client\FactFinderNg
+
+paths:
+  tests: .
+  data: ../../../_data
+  support: _support
+  log: ../../../_output
+  output: ../../../_output
+
+coverage:
+  enabled: true
+  remote: false
+  whitelist: { include: ['../../../../src/*'] }
+
+suites:
+  Service:
+    path: .
+    class_name: FactFinderNgTester
+    modules:
+      enabled:
+        - Asserts
+        - \PyzTest\Shared\Testify\Helper\Environment
```

Comment:
do we have it only on project level? Not sure if it's good idea to use helper from there

Reply:
We use it for every integration, I don't see Core level of this helper